autobahn/wamp_session.hpp

```diff
@@ -228,15 +240,22 @@ class wamp_session : public std::enable_shared_from_this<wamp_session<IStream, O
             const provide_options& options = provide_options());
 
     /*!
-     * Function called by the session when authenting. It always has to be re-implemented (if authentication is part of the system).
+     * Function called by the session when authenting. It always has to be
```

Comment:
Likely not your typo, but if you're already reflowing these comments then you might as well change `authenting` to `authenticating`.